docker/production/Dockerfile.cron

```diff
@@ -0,0 +1,32 @@
+FROM registry.fedoraproject.org/fedora:38
+MAINTAINER copr-devel@lists.fedorahosted.org
+
+ENV ENV=production
+
+RUN dnf -y update
+RUN dnf -y install git
```

Comment:
Same thing as @xsuchy said above. I recommend doing it the same way as in `Dockerfile.website`.